package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sails-db-migrate",
-  "version": "0.8.0",
+  "version": "0.8.1",
```

Comment:
You don't need to bump the version number. I'll do it when I publish the release.